woocommerce/woocommerce-android
# Shipping labels: Address suggestions

Implements #3019. 

Unit tests to be added in a subsequent PR.

![image](https://user-images.githubusercontent.com/1522856/106030251-4873a500-60ce-11eb-8a71-c9be26f87a30.png)

**To test:**
1. Prepare an order on the web that satisfies SL creation preconditions and make sure the origin and shipping addresses are valid (but not yet validated)
2. Go to Orders -> Open the order
2. Tap on Create shipping label button
3. Tap on the Continue button
4. Notice that the address suggestion screen is displayed, showing the original and a suggested address
5. Select the entered address and tap on Edit selected address button
6. Notice the address edit screen is displayed with the address
7. Tap on Done toolbar button
8. Notice the suggestion screen is shown again
9. Select the suggested address
10. Tap on Use selected address button
11. Notice the address is accepted and displayed in the creation flow
